docs/getting_started.md

```diff
@@ -228,7 +228,7 @@ spec:
       datacenter: SDDC-Datacenter
       datastore: DefaultDatastore
       folder: vm
-      resourcePool: Resources
+      resourcePool: 'Resources'
```

Comment:
Hi @andrewsykim,

If you're going to correct this then you may as well correct it all the way and make it `*/Resources`.

Reply:
Was just following what was in the existing examples, should I update the original one too then?

Reply:
Sure. I should have been more clear in our conversation the other day. Please see https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/403#issuecomment-510276822 for more information.